packages/webdriver/src/command.ts

```diff
@@ -14,10 +14,20 @@ export default function (
     method: string,
     endpointUri: string,
     commandInfo: CommandEndpoint,
-    doubleEncodeVariables = false
+    doubleEncodeVariables = false,
+    maskingPatterns: string[] = []
 ) {
     const { command, deprecated, ref, parameters, variables = [], isHubCommand = false } = commandInfo
 
+    // TODO dprevost to move to a better place!
+    const maskingRegExps = maskingPatterns.map((regexString) => {
+        if (!regexString.startsWith('/')) {return new RegExp(regexString)}
+        const lastSlashIndex = regexString.lastIndexOf('/')
+        const pattern = regexString.slice(1, lastSlashIndex)
```

Comment:
I guess we assume here that the regex starts with `/` should we double check for that?

Reply:
I wanted to use the regExp literally, but it gets lost when starting the runner somewhere. If I recall correctly, something triggered the run on a "message" emitted event, so I had to use string instead.

The first line uses RegExp patterns without the starting slash. This part assumes we have the two slashes. I can add more validation for sure. Also, I think I forgot to test it without flags; I will do so!

Reply:
I covered cases with slashes, without slashes, with flags, and without flags. I also added unit tests, and I moved that conversion into the utils file!
That should do the job; just unresolved the comment if I missed something!